### DISCUSSION TITLE
p-0 don't work

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1991] [Q&A] [answered]
### Description

I want to make a picture as background without padding 
this seems doesn't work
```py
ui.image('https://pic3.zhimg.com/v2-61ee09fa9f2fd830934ff282386d5cd2_r.jpg?source=1940ef5c') \
    .classes('w-full h-[calc(100vh-2rem)] p-0!important')
```

```py
ui.image('https://pic3.zhimg.com/v2-61ee09fa9f2fd830934ff282386d5cd2_r.jpg?source=1940ef5c') \
    .classes('w-full h-[calc(100vh-2rem)]').style('padding: 0!important')
```

there is always some space
![微信截图_20231109214857](https://github.com/zauberzeug/nicegui/assets/25096045/aaade164-ff77-4027-821f-edb652e300fa)

**Comment by v479038280** [comment, marked as answer]
the class fullscreen works

**Comment by falkoschindler** [comment]
The white gap is not caused by the image, but by the page content, i.e. the root element of the layout structure. So your question is basically answered in #499. But let me give an updated answer for version 1.4.x:

You can remove the padding like this
```py
from nicegui import context, ui

context.get_client().content.classes('p-0')
ui.image('https://picsum.photos/id/377/640/360')
```

like this
```py
image = ui.image('https://picsum.photos/id/377/640/360')
image.client.content.classes('p-0')
```

or like this:
```py
ui.query('.nicegui-content').classes('p-0')
ui.image('https://picsum.photos/id/377/640/360')
```

Same with `.style('padding: 0')`.

As a side note, `p-0!important` is no valid Tailwind class. To mark a class as "important", use a leading exclamation mark: `!p-0`. But in your case you don't need it anyway, because images come with zero padding by default.

**Reply by v479038280** [reply]
not import ,just  some test 
`ui.query('.nicegui-content').classes('p-0 !no-border-radius !rounded-none')`
the radius doesn't disapper

**Reply by falkoschindler** [reply]
What radius? The `content` has no radius.

**Reply by v479038280** [reply]
![微信截图_20231109233611](https://github.com/zauberzeug/nicegui/assets/25096045/34cdb207-da22-413f-b17a-d2f53cd0091f)

**Reply by falkoschindler** [reply]
We already discussed this matter in #1552 and concluded it's probably an Edge "feature".

**Reply by v479038280** [reply]
yes,I know that,
